examples/entry.py

```diff
@@ -0,0 +1,46 @@
+"""
+ Shows three entries :
+ - a simple entry
+ - a password entry
+ - a search entry
+
+"""
+
+from pylibui.core import App
+from pylibui.controls import (Window, Entry, SearchEntry, PasswordEntry,
+                              VerticalBox)
+
+
+class MyWindow(Window):
+
+    def onClose(self, data):
+        super().onClose(data)
+        app.stop()
+
+
+class MyEntry(Entry):
+
+    def onChanged(self, data):
+        print('entry changed!')
+
+
+app = App()
+
+window = MyWindow('Entry example')
+window.setMargined(10)
+
+entry = MyEntry()
+search_entry = SearchEntry()
+password_entry = PasswordEntry()
+
+vbox = VerticalBox()
+vbox.setPadded(10)
```

Comment:
Yes, is it a mistake. I have to correct this

By the way, I think we should replace the occurrences of ".setMargined(1)" by ".setMargined(True)", and so on (as always, only in the pythonic wrapper). I'll open an issue for that.